### ISSUE TITLE
Can not start Instant Preview with Xperia XZ2 Premium

### ISSUE THREAD

**Issue by TakuKobayashi** [google-ar/arcore-unity-sdk#390]
I confirmed that Xepria XZ 2 premium was newly added to the supported devices of ARCore.
https://developers.google.com/ar/discover/supported-devices
I could start ARCore applications with my Xepria XZ 2 premium, but could not start Instant Preview.(The detail is I could start the Instant Preview, but the camera was not activate.)
Are there a nice way to do it?

**Comment by fredsa** [comment]
Hi @TakuKobayashi 

1. What operating system are you developing on?
2. Does the Instant Preview app install correctly? Does it run? What does the app say after you hit Play in the editor?

**Comment by TakuKobayashi** [comment]
Hi @redsa
Thank you for replying, I will answer the question.
1. Android version is 8.0 and security patch level is August 2018. This is my device's screenshot.(I am sorry but my default language is Japanese. So, it is written by japanese.)
![screenshot_20181001-061551](https://user-images.githubusercontent.com/2100980/46272710-df051b00-c541-11e8-9782-3ccdadb42b71.png)

2. Yes, the Instant Preview app install correctly. And it runs.The following is the log output when the Instant Preview app is executed.
```
Instant Preview version 1.0.9
To disable Instant Preview, uncheck 'Instant Preview Enabled' under 'Edit > Project Settings > ARCore'.
```
There is nothing error logs.
And also, the Instant Preview app's permissions are accepted, like this.
![screenshot_20181001-061143](https://user-images.githubusercontent.com/2100980/46273200-0bba3200-c544-11e8-9052-559d0182f25d.png)
And this is the device's screenshot when the Instant Preview app is executed.
![screenshot_20181001-060824](https://user-images.githubusercontent.com/2100980/46273336-9733c300-c544-11e8-9bb8-cbd380294c29.png)

**Comment by pablisho** [comment]
Hi, I believe this issue was fixed with ARCore 1.5.0. Please update your SDK version. Feel free to comment if you still experience the issue after that.
